src/components/inputs/money-input/money-input.js

```diff
@@ -149,23 +149,42 @@ const createCurrencySelectStyles = ({
 //  "fractionDigits": 7
 // }
 // which equals 0.0123456 €
-
-// Parsing
-// Since most users are not careful about how they enter values, we will parse
-// both `.` and `,` as decimal separators.
+// when locale is provided we use the number 9999.999 becauause it'll either be formated to
+// A: 9,999.999
+// B: 9.999,999
+// otherwise, we infere the throwaway, separator from the amount
```

Comment:
👆Can you document the part above a bit better? It's a bit hard to understand...

Reply:
I've added step by step inline comments in the function itself

Reply:
So do we need this comment then? I find it confusing to be honest. I suppose it's clear for you that are familiar with the topic but we should try to make it understandable for everyone